### PR TITLE
Review process definition and instance details views

### PR DESCRIPTION
**Pull request by cristianonicolai**
Remove Errai warnings regarding wrong element between UI and view binding
Remove duplicated SLA text from instance details
Refactor view classes to hot expose widget elements
Replace widgets with Elemental2 elements
Remove unnecessary abstract classes

### CODE REVIEW

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsViewImpl.java [hunk 1]
```diff
@@ -13,157 +13,181 @@
...
+
+            instanceDetailsTab = GWT.create(TabListItem.class);
+            instanceDetailsTab.setText(constants.Process_Instance_Details());
+            instanceDetailsTab.setDataTargetWidget(instanceDetailsPane);
+            instanceDetailsTab.addStyleName("uf-dropdown-tab-list-item");
+
+            tabContent.add(instanceDetailsPane);
+            navTabs.add(instanceDetailsTab);
+        }
+
```
Comment (bsremac) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsViewImpl.java]: Maybe you don't need additional spaces for this.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/ProcessDefinitionDetailsTabViewImpl.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        deploymentIdLabel.textContent = constants.Deployment_Name();
+        humanTasksListLabel.textContent = constants.Human_Tasks();
+        usersGroupsListLabel.textContent = constants.User_And_Groups();
+        subprocessListLabel.textContent = constants.SubProcesses();
+        processDataListLabel.textContent = constants.Process_Variables();
+        processServicesListLabel.textContent = constants.Services();
+    }
+
+    @Override
+    public void setNroOfHumanTasksText(final String text) {
```
Comment (bsremac) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/ProcessDefinitionDetailsTabViewImpl.java]: A bit weird cutting :)
I would go with the whole word. It deosn't add much in length.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsViewImpl.java (hunk 1 above)
Comment (cristianonicolai) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsViewImpl.java]: Done

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/ProcessDefinitionDetailsTabViewImpl.java (hunk 2 above)
Comment (cristianonicolai) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/ProcessDefinitionDetailsTabViewImpl.java]: Renamed to numberOf

**Pull request merged**